example/package.json

```diff
@@ -55,11 +55,12 @@
     "@cumulus/sf-sns-report": "1.14.4",
     "@cumulus/sync-granule": "1.14.4",
     "@cumulus/test-processing": "1.14.4",
-    "aws-sdk": "^2.227.1",
+    "aws-sdk": "^2.585.0",
     "child-process-promise": "^2.2.1",
     "lodash.differencewith": "^4.5.0",
     "lodash.isequal": "^4.5.0",
-    "p-retry": "^2.0.0"
+    "p-retry": "^2.0.0",
+    "ssh2-streams": "^0.4.8"
```

Comment:
Where is this package used?

Reply:
The package has a sub dependency that is too permissive and un-upgradeable.  ssh2-streams sub .... .4? results in a buffer constructor deprecation warning under node 10 and later.